schema_poc/cherrypicked_samples_v1.avsc

```diff
@@ -0,0 +1,25 @@
+{
+  "namespace": "uk.ac.sanger.psd",
+  "type": "record",
+  "name": "CherrypickedSamplesMessage",
+  "doc": "A message notifying the cherrypicked state of a list of samples.",
+  "fields": [
+    {
+      "name": "samples",
+      "doc": "An array of samples that have been cherrypicked.",
+      "type": {
+        "type": "array",
+        "items": {
+          "name": "Sample", "type": "record",
+          "fields": [
+            {
+              "name": "sampleUuid",
+              "doc": "Unique sample UUID.",
+              "type": {"name": "version4SampleUuid", "type": "fixed", "size": 36}
+            }
+          ]
+        }
```

Comment:
What happens if we have to send this message long after it has been cherrypicked (like the next day) because of service downtime? Do we have a way of specifying the time of cherrypicking for a sample?

Reply:
I don't think we do, no.  However schema definitions were part of https://github.com/sanger/crawler/issues/473 which was completed weeks ago so if we'd like to review decisions made there, it has nothing to do with this PR and we should open a new ticket.  The schemas in the repo here will not be used.  The ones in Redpanda will be.